### DISCUSSION TITLE
Noooooooooo don't die

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [ExelentOS/ExelentOS-iso#7] [General]
I wanna be an ExelentOS user-or at least know how you got arch to use Plasma 5 over Plasma 6....

**Comment by thayeeboi890** [comment]
i could continue development
i just need more people to contribute
this os was and still is a one man project and cannot continue like that

also if you want plasma 5, you could install arch and also install the "downgrade" tool
then just ```sudo downgrade plasma-desktop```
and choose version 5

**Reply by neurodiverseEsoteric** [reply]
Ah, ok. Thanks. Still rooting for this OS!
